app/views/topics/index.html.erb

```diff
@@ -47,8 +47,8 @@
 
 <div>
   <% if user_signed_in? %>
-    <span class="hidden-xs pull-right"><%= link_to t(:open_new_discussion, default: "Open Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
+    <span class="hidden-xs pull-right"><%= link_to t(:start_discussion, default: "Start Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
```

Comment:
Great, okay - understood. Agree it's good to avoid adding new keys if at all possible - I should have looked more closely before commenting, sorry.

I now see your reasoning for specifying the "Start Discussion" text everywhere too, and it makes sense. I probably still wouldn't do it, because - especially in this case - the key matches the text, so it's not a big leap to see what it'll say (and it'll need updating everywhere if we change the text, or else it'd then be misleading), but happy with it whichever way you like :)